### PR TITLE
[docs] Update migrating to Xcode tutorial

### PR DESCRIPTION
**Pull request by BalestraPatrick**
As [Tulsi](https://github.com/bazelbuild/tulsi) has been officially archived, this PR updates the Bazel documentation to replace its mentions in favour of [rules_xcodeproj](https://github.com/buildbuddy-io/rules_xcodeproj). On top of that, this PR also removes the page that shortly documented how to move CocoaPods dependencies in favour of a short paragraph about experimental rules for the Swift Package Manager.

### CODE REVIEW

File: site/en/migrate/xcode.md [hunk 1]
```diff
@@ -22,27 +22,12 @@ troubleshooting solutions to address common errors.
...
-        of the project's source code.
-
-    *   Static analysis, address sanitizers, and thread sanitizers might not
-        work, since Bazel does not produce the outputs that Xcode expects for
-        those features.
-
-    *   If you generate an Xcode project with Tulsi and use that project to run
-        tests from within Xcode, Xcode will run the tests instead of
-        Bazel. To run tests with Bazel, run the `bazel test` command manually.
+    [rules_xcodeproj](https://github.com/buildbuddy-io/rules_xcodeproj/) whenever you
```
Comment (brentleyjones) [inline site/en/migrate/xcode.md:25]: Does this need `{: .external}`?

Comment (fweikert) [reply site/en/migrate/xcode.md:25]: Yes, good catch!

Comment (BalestraPatrick) [reply site/en/migrate/xcode.md]: Added now!

**Review by brentleyjones — APPROVED** [review]

**Comment by brentleyjones** [comment]
@fweikert (or whoever needs to approve this), anything we can do to get this merged? I would like to cherry-pick it into 6.1 so the docs website has it for that version.

**Review by gregestren — APPROVED** [review]

**Comment by brentleyjones** [comment]
@bazel-io flag

**Comment by kshyanashree** [comment]
@bazel-io fork 6.1.0

**Pull request closed without merging**